apps/zbugs/shared/queries.ts

```diff
@@ -57,13 +71,15 @@ export const queries = {
 
   user: syncedQueryWithContext(
     'user',
-    (_auth: AuthData | undefined, userID: string) =>
+    v.tuple([v.string()]).parse,
+    (_auth: AuthData | undefined, userID) =>
```

Comment:
Since auth is always `AuthData | undefined`, maybe there could be an API where you define this once? Eg.
```ts
const { syncedQueryWithContext } = createQueriesWithContextFactory<AuthData>()

syncedQueryWithContext(
	'user',
	validator.parse,
	// Automatically typed as AuthData | undefined
	(auth, userID) => {}
)
```
This should be quite easy to do in TS